decisions/log/0028-provide-clarity-on-characterization-factors.md

```diff
@@ -0,0 +1,55 @@
+# 28. Provide Clarity on Characterization Factors (AR5 and AR6)
+
+Date: 2024-01-16
+
+## Status
+
+Proposed
+
+## Context
+
+`characterizationFactors` is a mandatory property of the `CarbonFootprint` data type.
+It's data type is `String` and the value MUST be one of the following (see [4.2.2 Properties #characterizationFactors](https://wbcsd.github.io/tr/data-exchange-protocol/#element-attrdef-carbonfootprint-characterizationfactors)):
+- AR6
+  - for the Sixth Assessment Report of the Intergovernmental Panel on Climate Change (IPCC)
+- AR5
+  - for the Fifth Assessment Report of the IPCC
+
+However, some PCFs might be calculated with inputs that use both AR5 and AR6 as these can come from
+different sources (i.e., different actors in the supply chain). Since the IPCC will keep publishing
```

Comment:
I think it could be useful to mention here that the IPCC has generated 6 reports from 1990 to 2023, so an average cadence of one every 6-7 years, maybe more explicit language could be used to indicate the pace of change?